pkg/virt-launcher/virtwrap/api/schema_test.go

```diff
@@ -182,6 +182,31 @@ var exampleXMLppc64le = `<domain type="kvm" xmlns:qemu="http://libvirt.org/schem
   <iothreads>2</iothreads>
 </domain>`
 
+var _ = Describe("XML marshal of domain device", func() {
+	It("should not add user alias prefix to the name of a non-user-defined alias", func() {
```

Comment:
@EdDev Thank you for your review. Can you be specific on "it can just break additions..." ? 

You comment, "and as long as Alias is exported, we have a problem IMO.", here can be very interesting may point to an un-marked access to Alias.Name. Can you please elaborate a little?

Reply:
The suggested API changes the "default" to be **not** user-defined, while the current approach is the opposite.

Currently, we have usages of this form: `&api.Alias{Name: "foo"}` in the code-base. Some PR/s that are pending review/merge and future PR/s that contributors may look to propose may continue using this. If this change will get in, it may break any of those calls. One can argue that it should be catch by by the tests, but this is not assured always.

Reply:
@EdDev I am not sure I underrstand what you mean.

Reply:
I was able to make the `Alias::Name` private as `Alias::name`. This will invalidate any attempt to construct an Alias object via the *composition approach*. Any construction of Alias objects in new codes will be forced to use the `NewUserDefinedAlias()` constructor.